blocks/radio-input/__input/radio-input__input.css

```diff
@@ -0,0 +1,3 @@
+.radio-input__input:checked + .radio-input__label::after{
```

Comment:
done:)

Reply:
@wwily Great job! 

The only minor correction here is that we can skip adding a hover transition to `.radio-input__input:hover + .radio-input__label::before` since it is already set directly on the pseudo-element. This will help us to avoid redundancy in our CSS.